kickstarts/partials/post/db_init.ks.erb

```diff
@@ -4,7 +4,7 @@ cat > /bin/appliance-initialize.sh <<EOF
 [[ -d /var/opt/rh/rh-postgresql95/lib/pgsql/data/base ]] && exit 0
 [[ -s /etc/default/evm ]] && source /etc/default/evm
 echo "Initializing Appliance, please wait ..." > /dev/tty1
-appliance_console_cli --region 0 --internal --password smartvm
+appliance_console_cli --region 0 --internal --password smartvm --key
```

Comment:
@carbonin We can't do it here right?  Won't all appliances have the same key since this is run during kickstart?